### DISCUSSION TITLE
Why doesn't my MD document have a preview?

### DISCUSSION THREAD

**Discussion by lgcgo** [community/community#69377] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://github.com/lgcgo/letga-server/blob/main/docs/start_install.md

I have noticed that documents in other user docs have preview capability even if they are not README. How can I do this?

**Comment by ghost** [comment]


**Reply by lgcgo** [reply]
I want to know why the MD in my question does not have a preview, open it

**Reply by lgcgo** [reply]
Yeah, there are other people who find it strange too

**Reply by lgcgo** [reply]
It seems that this is not the reason. This problem existed before pags was set.

**Comment by Dipanshu089** [comment]
GitHub permits multiple Markdown files (README.md) in a repository, but only one is the default README displayed on the main repository page, usually the one in the root directory.

**Reply by lgcgo** [reply]
Please check this example, he did it
https://github.com/bufanyun/hotgo/blob/v2.0/docs/guide-zh-CN/start-installation.md

**Comment by seyLu** [comment]
#### Hi @lgcgo,

I've never had a problem like that, weird that preview shows up on a copy of letga-server README.md.
![image](https://github.com/community/community/assets/98249191/9201082c-5042-4150-8714-f5deee2004a1)

Can you try adding a `.editorconfig` file on the root dir of your repo (on VSCode you also need to install EditorConfig extension):
```editorconfig
# EditorConfig is awesome: https://EditorConfig.org

# top-most EditorConfig file
root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true

# Matches multiple files with brace expansion notation
# Set default charset
[*.{js,py}]
charset = utf-8

# 4 space indentation
[*.py]
indent_style = space
indent_size = 4

# Tab indentation (no size specified)
[Makefile]
indent_style = tab

# Indentation override for all JS under lib directory
[lib/**.js]
indent_style = space
indent_size = 2

# Matches the exact files either package.json or .travis.yml
[{package.json,.travis.yml}]
indent_style = space
indent_size = 2
```

**Comment by iprattham** [comment]
GitHub allows users to create multiple README.md files in a repository, but only your one README will get displayed on the main repository page which will be you default one i.e. README.md, usually the one in the root directory as all other readme's will be of diff names and can;t be of the same name.

**Reply by adityasharmaa09** [reply]
Yeah, that's true! I faced the same issue, but now solved thankyou.

**Comment by zhangt2333** [comment]
@lgcgo 

The `linguist-language` part in `.gitattributes` make GitHub cannot recognise the current file as Markdown.

https://github.com/lgcgo/letga-server/blob/ae9a8117d7797b6031685be33a19bc488fb88af6/.gitattributes

**Comment by queenofcorgis** [comment]
@lgcgo we're going through some old posts and want to check if you got this solved, if you did feel free to share the solution or mark one of the folks' responses above as the answer.
